RevenueCatUI/Templates/V2/Components/Carousel/CarouselComponentView.swift

```diff
@@ -13,19 +13,22 @@
 // swiftlint:disable file_length
 
 import Foundation
-import RevenueCat
+@_spi(Internal) import RevenueCat
```

Comment:
I think the `@_spi(Internal)` is not needed for now
```suggestion
import RevenueCat
```